ocaml/xapi/test_vdi_allowed_operations.ml

```diff
@@ -294,19 +294,37 @@ let all_cbt_operations = [`enable_cbt; `disable_cbt; `data_destroy] in
 
         (fun vdi -> pass_data_destroy vdi;
           Db.VDI.set_cbt_enabled ~__context ~self:vdi ~value:false
-        ) ,         Some (Api_errors.vdi_no_cbt_metadata, []) ;
+        ) ,         Some (Api_errors.vdi_no_cbt_metadata , []) ;
 
         (fun vdi -> let vM =
                       Test_common.make_vm ~__context () in
           let _: _ API.Ref.t = Test_common.make_vbd ~__context ~vDI:vdi ~vM ~currently_attached:true () in
           pass_data_destroy vdi
-        ) ,         Some (Api_errors.vdi_in_use, []) ;
+        ) ,         Some (Api_errors.vdi_in_use , []) ;
```

Comment:
This is a really minor thing, but usually we try to remove irrelevant whitespace changes to minimize the diff that people look at. There is no need to change this PR though, this is just for future ones.